src/views/v2/tickets.php

```diff
@@ -72,6 +72,15 @@ class="tribe-tickets"
 
 	</form>
 
-	<?php $this->template( 'v2/modal/target' ); ?>
-
+	<?php
+	/**
+	 * Allows injection of additional markup after the form tag but within the div of this template.
+	 *
+	 * @since TBD
+	 *
+	 * @see  Tribe__Template\do_entry_point()
+	 * @link https://docs.theeventscalendar.com/reference/classes/tribe__template/do_entry_point/
+	 */
+	$this->do_entry_point( 'after_form' );
```

Comment:
Do we have any type of convention on how to prefix entry points, if we have to do it or not? So far I been using `rsvp_` or `tickets_`. I know that then the file path will be part of the hook, but wanted to check if we should think about some consistency of the entry point naming in general, as we already have a couple since RSVP v2.

Reply:
From what I understand of entry point names, they are just unique to the view itself. Not to all entry points. Prefixing seemed like an unnecessary thing. Unless you see differently there, maybe we can also ask @bordoni